lib/AutoDragSortableView.js

```diff
@@ -589,7 +592,10 @@ export default class AutoDragSortableView extends Component{
     render() {
         return (
             <ScrollView
-                bounces={false}
+                bounces={this.props.onRefresh}
+                refreshControl={this.props.onRefresh && (
+                  <RefreshControl onRefresh={this.props.onRefresh} refreshing={this.props.refreshing} enabled={this.state.scrollEnabled} />
```

Comment:
Disable RefreshControl when scrollEnabled is false because that means we are processing a drag of an item. The drag can conflict with the pull to refresh gesture if it is left enabled.